dev/com.ibm.ws.classloading/src/com/ibm/wsspi/library/Library.java

```diff
@@ -69,4 +70,8 @@
      */
     Collection<ArtifactContainer> getContainers();
 
+    /**
+     * Get the allowed SPI types for this shared library.
+     */
+    EnumSet<SpiType> getSpiTypeVisibility();
```

Comment:
Same beta question, if this really is SPI

Reply:
Answered above.  All beta SPI operations are fenced and will no-op when invoked in a non-beta edition.